R/REST_helpers.R

```diff
@@ -36,12 +36,12 @@ sbtools_POST <- function(url, body, session, timeout = default_timeout(), ...){
 #' @keywords internal
 sbtools_GET <- function(url, ..., session, timeout = default_timeout()){
 	
-	supported_types <- c('text/plain','application/json')
+	supported_types <- c('text/plain','application/json','application/x-gzip')
```

Comment:
@lawinslow not sure, doesn't seem like we should block `text/html` (see above), but it would be good to have a decent error when you run up against sb permissions and fail.